### ISSUE TITLE
Update git to latest

### ISSUE THREAD

**Issue by jerone** [ForkIssues/TrackerWin#590]
Azure DevOps is giving the following warning when pushing to remote:

```
$ git push --progress --verbose origin refs/heads/release/sprint-39:refs/heads/release/sprint-39 -u

Total 0 (delta 0), reused 0 (delta 0)
POST git-receive-pack (179 bytes)
remote: We noticed you're using an older version of Git. For the best experience, upgrade to a newer version.        
Pushing to https://dev.azure.com/organization/project/_git/repo
To https://dev.azure.com/organization/project/_git/repo
 * [new branch]      release/sprint-39 -> release/sprint-39
updating local tracking ref 'refs/remotes/origin/release/sprint-39'
```

Correctly, Azure DevOps warns about the git version being out of sync. Yes I know, in the settings you can switch to an custom git instance. But as the default is to use Fork internal git instance, I would suggest to update the internal git version.

Fork runs on git **2.33.0.windows.1**
The [latest](https://git-scm.com/download/win) git is **2.24.1.windows.2**

**Comment by DanPristupov** [comment]
The reason for that warning is the vulnerability (https://lwn.net/Articles/806972/) in git 2.24.0 (and 2.23.0 too).

Regarding 2.24.1: it's not even on git-scm.com yet. UPD: already on git-scm, but only for the Windows platform yet.

**Comment by jerone** [comment]
Yeah, [multiple security fixes](https://github.com/git-for-windows/git/releases/tag/v2.24.1.windows.2).

What is the plan for updating the git instance inside Fork? Do you have a fixed plan, or just when you have time to update?

**Comment by DanPristupov** [comment]
I just released the 1.0.42.**6** hotfix. It contains git 2.24.1.
